src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -58,6 +58,11 @@ public ReadOnlyPerson getPerson() {
     }
 
     @Override
```

Comment:
Add a header comment for your overriden methods. It is necessary for other people to understand what the overriden method is for.